routes.js

```diff
@@ -180,4 +180,9 @@ module.exports = function (app, passport) {
                 throw error
             })
     })
+
+    // todo check on production ,currently not sure how to check..
```

Comment:
isn't it enough to do build + sever on NODE_ENV=production + using 3001 instead of 3000 
to test?